### DISCUSSION TITLE
v19.22.6-4.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#173] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.22.6`  
Current YTLite Version: `4.0.1`  
Current App Compatibility: `iOS/iPadOS 15.0` or later

**RELEASE 19.22.6-4.0.1:** (2024-06-09)

- Updated Youtube from `19.21.3` to `19.22.6`
- Updated submodules
- Updated YTLite from `4.0` to `4.0.1`
    - Added ability to play videos in external players (System, Infuse, VLC)

    - Added option to keep Shorts videos in the Subscriptions tab (https://github.com/dayanch96/YTLite/issues/91)

    - Added ability to open shorts as regular videos using YTPlus menu button in Shorts

    - Added ability to enable/disable auto-skip shorts option using YTPlus menu button in Shorts

    - Added option to disable Autoplay end-screen recommendation (Enabling this option will also disable autoplay next video feature)

    - Improved Shorts downloading which directly downloads best mp4 video

    - Improved YTPlus button compatibility with the latest YouTube versions

    - Improved YTPlus button compatibility with Inline muted playback (Player that displayed on video thumbnail)

    - Fixed blank space issues on the big screens (https://github.com/dayanch96/YTLite/issues/105)

    - Fixed crashes when trying to show share sheet on iPad devices

    - Updated localization strings (https://github.com/dayanch96/YTLite/issues/106)

    - Corrected some localization strings (https://github.com/dayanch96/YTLite/issues/112)

    - Fixed an issue that caused the search bar to disappear (https://github.com/dayanch96/YTLite/issues/117)
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.22.6-4.0.1'>v19.22.6-4.0.1 - YTLitePlus</a>.</em>

**Comment by tha1kat** [comment]
you left the ipa lil bro

**Comment by CSummerChild** [comment]
Hey it's been 19 hours and the ipa is still able to be downloaded

**Reply by Balackburn** [reply]
> Can I build? I get an error

Fixed

**Reply by Joshua-18** [reply]
I still can’t build the ipa

**Reply by Balackburn** [reply]
> @Balackburn I can't build again...

Yes their is an issue I don't understand (it work locally) where YTLITE_VERSION result in "null" instead of the version, it break the building I though this would be easy to fix, working on it

**Comment by Joshua-18** [comment]
Is it not possible to hide ui elements from shorts anymore?

**Reply by ghost** [reply]
I don't know if it's impossible yet, I think it's being investigated.

**Reply by Joshua-18** [reply]
Does the YTPlus menu button crashes for you also?
